src/gason.cpp

```diff
@@ -1,8 +1,8 @@
 #include "gason.h"
 #include <stdlib.h>
 
-#define JSON_ZONE_SIZE 4096
-#define JSON_STACK_SIZE 32
+constexpr size_t JSON_ZONE_SIZE = 4096;
+constexpr size_t JSON_STACK_SIZE = 32;
```

Comment:
If you do so, you would have to cast them in the allocator. It is just a warning, but good source code does not generates warnings.

The parser compares this value to signed integers and the allocator to unsigned integers.

When you use a macro, it is not an int ! It is a integer literal which behavior depends on how you use it in the code. This demonstrates why macros are bad practices.